deployment/docker-compose/docker-compose.yml

```diff
@@ -31,6 +33,8 @@ services:
     restart: always
     env_file:
       - .base.env
+    ports:
```

Comment:
You don't need to expose these ports since they on the same virtual network. And traffic is going through Caddy